docs/docs/statistics/power.mdx

```diff
@@ -0,0 +1,184 @@
+6. By clicking "Edit", you can toggle [sequential testing](/statistics/sequential) on and off to compare power. Enabling sequential testing decreases power.
+7. You can alter the number of variations in your experiment. Increased variations result in smaller sample sizes per variation, resulting in lower power.
+
+#need to update this Image, go through results interpretation
+
+## How do I interpret power analysis results?
+
+Below is an example power analysis results page.
+![User interface for quantile metrics](/images/statistics/power_results.png)
+
+GrowthBook includes both power and MDE results to ensure that customers comfortable with either tool can use them to make decisions.
+
+## How should I pick my effect size?
+
+Selecting your effect size for power analysis requires careful thought. Your effect size is your anticipated metric lift due to your feature. Obviously you do not have complete information about the true lift, otherwise you would not be running the experiment!
+
+We advocate running power analysis for multiple effect sizes. The following questions may elicit helpful effect sizes:
+
+1. What is your best guess for the potential improvement due to your feature? Are there similar historical experiments, or pilot studies, and if so, what were their lifts?
+2. Suppose your feature is amazing - what do you think the lift would be?
+3. Suppose your feature impact is smaller than you think - how small could it be?
+
+Ideally your experiment has high power (see [here](#what-is-a-high-powered-experiment)) across a range of effect sizes.
+
+## What is a high-powered experiment?
+
+In clinical trials, the standard is 80%. This means that if you were to run your clinical trial 100 times with different patients and different randomizations each time, then you would observe statistically significant results in at least roughly 80 of those trials. When calculating MDEs, we use this default of 80%.
+
+That being said, running an experiment with less than 80% power can still help your business. The purpose of an experiment is to learn about your business, not simply to roll out features that achieve statistically significant improvement. The biggest cost to running low-powered experiments is that
+
+## How can I increase my experimental power?
+
+You can increase experimental power by increasing the number of daily users, perhaps by either expanding your population to new segments, or by including a larger percentage of user traffic in your experiment.
+
+Similarly, if you have more than 2 variations, reducing the number of variations increases power.
+
+## FAQ
+
+Frequently asked questions:
+
+1. How do I pick the number for Estimated users per week? If you know your number of daily users, you can multiply that by 7. If traffic varies by day of the week, you may want to do something like (5 _ average weekday traffic + 2 _ average weekend traffic) / 7.
+
+## GrowthBook implementation
```

Comment:
Is there a way to separate this into a simpler section for the high level summary for what each quantity is, and then an "in-depth" section that talks more about the specifics? This is really low priority ask though, so feel free to ignore for now.

Reply:
I think it might help to chat quickly about what you are looking for.  I'll make a note to bring this up next week, and update the docs if needed.